frontend/src/patientApp/timeOfTest/__snapshots__/TestResult.test.tsx.snap

```diff
@@ -65,9 +65,7 @@ exports[`TestResult should show a negative result 1`] = `
         </h2>
         <p
           class="margin-top-05"
-        >
-          Testing device
```

Comment:
The snapshot changes here seem incorrect (empty paragraph now rather than showing the test device)

Reply:
Aha - good catch, forgot to update the mock store key to `deviceTypeModel`, should be fixed now